lib/interpret_date.rb

```diff
@@ -2,6 +2,24 @@
 require "interpret_date/version"
 
 module InterpretDate
+  def self.included base
+    base.extend ClassMethods
+  end
+
+  module ClassMethods
+    def interpret_date_assignment_for(*attributes)
```

Comment:
What do you think about shortening this method name? Some possible alternatives:

`interpret_date_for`
`attr_interpret_date`
`interpret_date_attr`
`attr_writer_interpret_date`
`attr_interpret_date_writer`

I like the first one in the list, the others are just thoughts.